### DISCUSSION TITLE
Jupyterlab-myst extension with R-Kernel

### DISCUSSION THREAD

**Discussion by ro-bee** [executablebooks/meta#1157] [Q&A] [answered]
Hi, can anyone confirm if the myst extension for jupyterlab is compatible with any kernel or only Python?

I've tried it with R and it doesn't seem to work but always the possibility of user error!

<img width="235" alt="image" src="https://github.com/executablebooks/meta/assets/82892736/3d0d8315-979d-4401-9d92-632c7610d265">

**Comment by agoose77** [comment, marked as answer]
Currently the underlying R kernel does not support the mechanism that we use to evaluate expressions (c.f. https://github.com/IRkernel/IRkernel/issues/678). We can look to try and allocate some resources towards fixing that, but I can't be certain! I'll loop back here.

**Reply by ro-bee** [reply]
Thank you for confirming (glad to know its not user error!) - would be awesome if that could be resolved given most of the health data science research we do uses R and I'd love to be able to create scientific publications directly from Jupyterlab. For now (and for anyone else perhaps searching) the only solution I'm aware of is using `python-markdown` with the classic JNBs. If there are any other known R options, would love to know about them

**Reply by agoose77** [reply]
c.f. https://github.com/IRkernel/IRkernel/pull/742
